ui/src/PaperSummary.tsx

```diff
@@ -23,6 +23,11 @@ function warnOfUnimplementedActionAndTrack(actionType: string) {
   }
 }
 
+function goToLibrary() {
+  const s2LibraryUrl = `https://www.semanticscholar.org/me/library`;
```

Comment:
small style comment: could be single quotes rather than backticks as there's no variable substitution.